modules/monitoring-supported-remote-write-authentication-settings.adoc

```diff
@@ -0,0 +1,314 @@
+            key: secret <3>
+          profile: <AWS_profile_name> <4>
+          roleArn: <AWS_role_arn> <5>
+----
+<1> The AWS region.
+<2> The name of the `Secret` object containing the AWS API access credentials.
+<3> The AWS API access credentials contained in the specified `Secret` object.
+<4> The name of the AWS profile that is being used to authenticate.
+<5> The unique identifier for the Amazon Resource Name (ARN) assigned to your role.
+
+
+.Sample YAML for basic authentication
+
+The following shows sample basic authentication settings for a `Secret` object named `rw-basic-auth` in the `openshift-monitoring` namespace: 
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rw-basic-auth
+  namespace: openshift-monitoring
+stringData:
+  user: <basic_username> <1>
+  password: <basic_password> <2>
+type: Opaque
+----
+<1> The username.
+<2> The password.
+
+The following sample shows a `basicAuth` remote write configuration that uses a `Secret` object named `rw-basic-auth` in the `openshift-monitoring` namespace.
+It assumes that you have already set up authentication credentials for the endpoint. 
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://basicauth.example.com/api/write"
+        basicAuth:
+          username:
+            name: rw-basic-auth <1>
+            key: user <2>
+          password:
+            name: rw-basic-auth <1>
+            key: password <3>
+----
+<1> The name of the `Secret` object that contains the authentication credentials.
+<2> The username contained in the specified `Secret` object.
+<3> The password contained in the specified `Secret` object.
+
+.Sample YAML for bearer token authentication using a `Secret` Object
+
+The following shows bearer token settings for a `Secret` object named `rw-bearer-auth` in the `openshift-monitoring` namespace:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rw-bearer-auth
+  namespace: openshift-monitoring
+stringData:
+  token: <authentication_token> <1>
+type: Opaque
+----
+<1> The authentication token.
+
+The following shows sample bearer token config map settings that use a `Secret` object named `rw-bearer-auth` in the `openshift-monitoring` namespace:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://authorization.example.com/api/write"
+        authorization:
+          type: Bearer
+          credentials:
+            name: rw-bearer-auth <1>
+            key: token <2>
+----
+<1> The name of the `Secret` object that contains the authentication credentials.
+<2> The authentication token contained in the specified `Secret` object.
```

Comment:
```suggestion
<2> The key in the specified `Secret` object containing the authentication token.
```

Reply:
Edited per your suggestion as follows:

`<2> The key that contains the authentication token in the specified `Secret` object.
`